src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/PropertyExpression.cs

```diff
@@ -38,55 +38,55 @@ private void OnPropertyChanged()
 
             if (Property.Type.IsEnumEx())
             {
-                CreateDataTypeExpressionIfNotCompatible(() => new StringExpression());
+                DataTypeExpression = new StringExpression();
```

Comment:
The unit tests were failing. I have reverted (see latest version). It failed when assigning a custom expression and then setting the property. Then it overwrote the expression (which can cause the same issues during deserialization).